netci.groovy

```diff
@@ -317,17 +329,76 @@ class Constants {
     ]
   
     def static validLinuxArm64Scenarios = [ 
-        'normal',
-        'r2r',
-        'innerloop',
-        'gcstress0x3',
-        'gcstress0xc'
+               'innerloop',
+               'normal',
+               'r2r',
+               'gcstress0x3',
+               'gcstress0xc'
+    ]
+
+    // Note: no GCStress-related scenario is enabled currently.
+    def static validLinuxArmScenarios = [
+               'innerloop',
+               'normal',
+               'r2r',
+               'r2r_jitstress1',
+               'r2r_jitstress2',
+               'r2r_jitstressregs1',
+               'r2r_jitstressregs2',
+               'r2r_jitstressregs3',
+               'r2r_jitstressregs4',
+               'r2r_jitstressregs8',
+               'r2r_jitstressregs0x10',
+               'r2r_jitstressregs0x80',
+               'r2r_jitstressregs0x1000',
+               'r2r_jitminopts',
+               'r2r_jitforcerelocs',
+               // 'r2r_gcstress15',
+               'minopts',
+               'forcerelocs',
+               'jitstress1',
+               'jitstress2',
+               'jitstressregs1',
+               'jitstressregs2',
+               'jitstressregs3',
+               'jitstressregs4',
+               'jitstressregs8',
+               'jitstressregs0x10',
+               'jitstressregs0x80',
+               'jitstressregs0x1000',
+               'jitstress2_jitstressregs1',
+               'jitstress2_jitstressregs2',
+               'jitstress2_jitstressregs3',
+               'jitstress2_jitstressregs4',
+               'jitstress2_jitstressregs8',
+               'jitstress2_jitstressregs0x10',
+               'jitstress2_jitstressregs0x80',
+               'jitstress2_jitstressregs0x1000',
+               'tailcallstress'
+               // 'gcstress0x3',
+               // 'gcstress0xc',
+               // 'zapdisable',
+               // 'heapverify1',
+               // 'gcstress0xc_zapdisable',
+               // 'gcstress0xc_zapdisable_jitstress2',
+               // 'gcstress0xc_zapdisable_heapverify1',
+               // 'gcstress0xc_jitstress1',
+               // 'gcstress0xc_jitstress2',
+               // 'gcstress0xc_minopts_heapverify1'
     ]
 
     def static configurationList = ['Debug', 'Checked', 'Release']
 
     // This is the set of architectures
-    def static architectureList = ['arm', 'armlb', 'x86_arm_altjit', 'x64_arm64_altjit', 'arm64', 'x64', 'x86']
+    // Some of these are pseudo-architectures:
+    //    armlb -- same as arm, but use the LEGACY_BACKEND JIT
+    //    armem -- ARM builds/runs using an emulator. Used for Ubuntu/Ubuntu16.04/Tizen runs.
```

Comment:
I might leave it as-is for now, in hopes that we'll simplify this slightly by eliminating the non-Tizen armem stuff soon. And because "armel" is already being used in some cases. I guess theoretically we might want "armel" and "armelem", with the latter being "armel run in an emulator".